flow_reorganize_symlinks/scaffold_gzip.cwl

```diff
@@ -12,6 +12,8 @@ arguments:
     #fasta file
     position: 1
     valueFrom: $(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_genomic_fasta[0])/$(inputs.genome_fasta_name[0])
+  - position: 2
```

Comment:
@HsiuKangHuang Can you remove the readme gzip step? the previous shell script did this, but it was a bug rather than a feature...